### PR TITLE
Test multisig upgrade (DO NOT MERGE)

### PR DESCRIPTION
**Pull request by encody**

### CODE REVIEW

File: workspaces-tests/tests/upgrade_multisig.rs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    );
+
+    let hello: String = alice
+        .view(contract.id(), "say_hello", vec![])
+        .await
+        .unwrap()
+        .json()
+        .unwrap();
+
+    assert_eq!(hello, "I am the second contract");
```
Comment (nearken) [inline workspaces-tests/tests/upgrade_multisig.rs:92]: It may be good to add an additional test or check that you can't call a function in the first contract

**Review by nearken — APPROVED** [review]
Approved suggestion to add additional test for all code paths and corner cases

**Comment by encody** [comment]
Techniques utilized in #84
